src/Nancy.Tests/Unit/ModelBinding/DefaultBinderFixture.cs

```diff
@@ -62,6 +62,40 @@ public void Should_throw_if_body_deserializers_is_null()
         }
 
         [Fact]
+        public void Should_throw_if_body_deserializer_fails_and_IgnoreErrors_is_false()
+        {
+            // Given
+            var deserializer = new ThrowingBodyDeserializer<FormatException>();
+            var binder = this.GetBinder(bodyDeserializers: new[] { deserializer });
+
+            var context = CreateContextWithHeader("Content-Type", new[] { "application/xml" });
+
+            var config = new BindingConfig { IgnoreErrors = false };
+
+            // When
+            var result = Record.Exception(() => binder.Bind(context, this.GetType(), null, config));
+
+            // Then
+            result.ShouldBeOfType<ModelBindingException>();
+            result.InnerException.ShouldBeOfType<FormatException>();
+        }
+
+        [Fact]
+        public void Should_not_throw_if_body_deserializer_fails_and_IgnoreErrors_is_true()
+        {
+            // Given
+            var deserializer = new ThrowingBodyDeserializer<FormatException>();
+            var binder = this.GetBinder(bodyDeserializers: new[] { deserializer });
+
+            var context = CreateContextWithHeader("Content-Type", new[] { "application/xml" });
+
+            var config = new BindingConfig { IgnoreErrors = true };

```

Comment:
You could extract these 4 setup lines from this test and the one above - what do you think? Going to far?